lib/env.js

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('./config');
+
+exports.isProdLike = function isProdLike() {
+  var env = config.get('env');
+  return env === 'prod' || env === 'stage';
```

Comment:
Oh right. fxa-dev will need an update to set a config of events since latest.dev.lcip.org runs as 'env === "stage"', and I guess an SQS (or maybe we can just dummy that on those boxes).